sympy/functions/elementary/complexes.py

```diff
@@ -696,16 +696,21 @@ def _eval_rewrite_as_conjugate(self, arg, **kwargs):
 
 
 class arg(Function):
-    """
-    returns the argument (in radians) of a complex number.  The argument is
-    evaluated in consistent convention with atan2 where the branch-cut is
-    taken along the negative real axis and arg(z) is in the interval
-    $(-\pi, \pi]$.  For a positive number, the argument is always $0$.
+    r"""
+    Returns the argument (in radians) of a complex number. The argument is
+    evaluated in consistent convention with ``atan2`` where the branch-cut is
+    taken along the negative real axis and ``arg(z)`` is in the interval
+    $(-\pi,\pi]$. For a positive number, the argument is always 0; the
+    argument of a negative number is $\pi$; and the argument of 0
+    is undefined and returns ``nan``. So the ``arg`` function will never nest
+    greater than 3 levels since at the 4th application, the result must be
+    nan; for a real number, nan is returned on the 3rd application.
```

Comment:
Is it necessary to say this?

Reply:
That is nothing that I have added on purpose (and wouldn't know), so probably a consequence of me starting from the wrong branch...

I'll see if I can get rid of it.

Reply:
That doesn't seem to be in the latest (after I fixed the merge conflicts), only some formatting of numbers to latex.